changelog.md

```diff
@@ -168,7 +168,12 @@
   cannot be applied to every use case. The limitations and the (lack of) reliability
   of `round` are well documented.
 
+- New type: `cuintptr_t` - the same as `uintptr_t` in C.
```

Comment:
Well this is a (minor) feature addition, if we backport that, we don't follow semver.

Suggestion: Use the type internally but don't export it.

Reply:
> Well this is a (minor) feature addition, if we backport that, we don't follow semver.

Then I'm declaring it a bug fix for this macOS error when trying to use Nim's `uint` instead of C's `uintptr_t`:

```text
/Users/travis/build/status-im/nim-libbacktrace/libbacktrace.nim:74:31: error: no matching function for call to 'get_debugging_info_c'

        struct debugging_info* T1_ = get_debugging_info_c((&programCounters->data[((NI) 0)]), maxLength);       struct debugging_info* functionInfoPtr = T1_;

                                     ^~~~~~~~~~~~~~~~~~~~

/Users/travis/build/status-im/nim-libbacktrace/libbacktrace_wrapper.h:42:24: note: candidate function not viable: no known conversion from 'NU *' (aka 'unsigned long long *') to 'uintptr_t *' (aka 'unsigned long *') for 1st argument

struct debugging_info *get_debugging_info_c(uintptr_t *program_counters, int max_length);

                       ^

/Users/travis/build/status-im/nim-libbacktrace/libbacktrace.nim:59:6: error: cannot initialize a variable of type 'NU *' (aka 'unsigned long long *') with an rvalue of type 'uintptr_t *' (aka 'unsigned long *')

        NU* T1_ = get_program_counters_c(maxLength, ((int) 2)); NU* pcPtr = T1_;

            ^     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

2 errors generated.

Error: execution of an external compiler program 'clang++ -c  -w -I/Users/travis/build/status-im/nim-libbacktrace -I/Users/travis/build/status-im/nim-libbacktrace/install/usr/include -g -std=gnu++14 -funsigned-char  -I/Users/travis/build/status-im/nim-libbacktrace/Nim/lib -I/Users/travis/build/status-im/nim-libbacktrace/tests -o /Users/travis/.cache/nim/test2_d/@m..@slibbacktrace.nim.cpp.o /Users/travis/.cache/nim/test2_d/@m..@slibbacktrace.nim.cpp' failed with exit code: 1
```

There, problem solved ;-)